prompt2model/utils/parse_json_responses.py

```diff
@@ -0,0 +1,104 @@
+"""Utility file for parsing OpenAI json responses."""
+import json
+from typing import Union
+
+import openai
+
+from prompt2model.utils import api_tools, get_formatted_logger
+from prompt2model.utils.api_tools import API_ERRORS, handle_api_error
+
+logger = get_formatted_logger("ParseJsonResponses")
+
+
+class JsonParsingFromLLMResponse:
```

Comment:
I feel like this doesn't need to be a class. Could we just use module-level functions and pass in `max_api_calls` to `extract_response` and `parse_prompt_to_fields`?